// @expo-google-fonts/nunito/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Nunito_200ExtraLight = require('./200ExtraLight/Nunito_200ExtraLight.ttf');
export const Nunito_300Light = require('./300Light/Nunito_300Light.ttf');
export const Nunito_400Regular = require('./400Regular/Nunito_400Regular.ttf');
export const Nunito_500Medium = require('./500Medium/Nunito_500Medium.ttf');
export const Nunito_600SemiBold = require('./600SemiBold/Nunito_600SemiBold.ttf');
export const Nunito_700Bold = require('./700Bold/Nunito_700Bold.ttf');
export const Nunito_800ExtraBold = require('./800ExtraBold/Nunito_800ExtraBold.ttf');
export const Nunito_900Black = require('./900Black/Nunito_900Black.ttf');
export const Nunito_200ExtraLight_Italic = require('./200ExtraLight_Italic/Nunito_200ExtraLight_Italic.ttf');
export const Nunito_300Light_Italic = require('./300Light_Italic/Nunito_300Light_Italic.ttf');
export const Nunito_400Regular_Italic = require('./400Regular_Italic/Nunito_400Regular_Italic.ttf');
export const Nunito_500Medium_Italic = require('./500Medium_Italic/Nunito_500Medium_Italic.ttf');
export const Nunito_600SemiBold_Italic = require('./600SemiBold_Italic/Nunito_600SemiBold_Italic.ttf');
export const Nunito_700Bold_Italic = require('./700Bold_Italic/Nunito_700Bold_Italic.ttf');
export const Nunito_800ExtraBold_Italic = require('./800ExtraBold_Italic/Nunito_800ExtraBold_Italic.ttf');
export const Nunito_900Black_Italic = require('./900Black_Italic/Nunito_900Black_Italic.ttf');
